packages/sdk/explorer/src/block-explorer.ts

```diff
@@ -53,7 +60,9 @@ export class BlockExplorer {
 
   constructor(private kit: ContractKit, readonly contractDetails: ContractDetails[]) {
     this.addressMapping = mapFromPairs(
-      contractDetails.map((cd) => [cd.address, getContractMappingFromDetails(cd)])
+      contractDetails
+        .filter((cd) => /Proxy/.exec(cd.name) == null)
```

Comment:
This will remove everything that contains Proxy. Even if it's not the case, But things like "ProxyConstructor", "ProxyChecker" or whatever will be excluded. With `/Proxy$/` at least we can check that it finishes with that word (which we are using as a convention) 

```suggestion
        .filter((cd) => /Proxy$/.exec(cd.name) == null)
```